buildc-p.sh

```diff
@@ -0,0 +1,19 @@
+# Check if both piece name and type are provided
+if [ -z "$1" ] || [ -z "$2" ]; then
+    echo "Please provide both a piece name and a type (c/community or cc/custom)."
+    exit 1
+fi
+
+# Check if the type is either 'c'/'community' or 'cc'/'custom'
+if [ "$1" = "c" ] || [ "$1" = "community" ]; then
+    type="community"
+elif [ "$1" = "cc" ] || [ "$1" = "custom" ]; then
+    type="custom"
+else
+    echo "Invalid type. Please use 'c/community' or 'cc/custom'."
+    exit 1
+fi
+
+# Run the build command with the provided piece name and type
+nx build pieces-$type-$2
+
```

Comment:
We have already fixed the pieces to remove the type from the name. I will change the name for this one before merging. 

There was an issue in CLI which has been fixed. I think this is no longer needed.

Can you please remove?

Reply:
The main purpose of this is to rebuild fast for testing purposes, it worked like this:
```
sh buildc-p.sh c wootric
```
c for community and cc for custom.

Let me know if deleting this is still needed